doc/blog/201808-UnoUnderTheHoodIntro.md

```diff
@@ -28,15 +28,15 @@ top of [WebAssembly](https://webassembly.org/).
 
 There are a few things that need to be done to get your app running: 
 
--   At compile time, Uno parses XAML files into C\# code, creating the
+-   > At compile time, Uno parses XAML files into C\# code, creating the
```

Comment:
Would be better to remove the quotation from this list.

Reply:
Done